example/lib/example.dart

```diff
@@ -76,3 +76,25 @@ Visible libraries: $visibleLibraries
     '.dart': ['.dart.info']
   };
 }
+
+class MovingBuilder implements Builder {
```

Comment:
It might be good to add some comments on this builder explaining what it does and why that is interesting as an example :).

Reply:
I've replaced it with a new builder that reads strings from somewhere in `assets/` and generates matching Dart constants in `lib/`. I think this may be closer to a practical use of this feature as well.